lib/BlockedBy.js

```diff
@@ -274,7 +274,8 @@ class BlockedBy extends NodeResque.Plugin {
             redisInstance: this.queueObject.connection.redis,
             buildId,
             status: 'BLOCKED',
-            statusMessage: `Blocked by these running build(s): ${blockingBuildIds}`
+            statusMessage: 'Blocked by these running build(s): ' +
```

Comment:
what does it look like if there's multiple buildIds?

Reply:
Oh .. I forgot in the case of multiple buildIds.
Therefore I will fix from now on.